### PR TITLE
Updated Retrievals requirement based on feedback at IETF 116

### PR DESCRIPTION
**Pull request by ykarandikar**

### CODE REVIEW

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ end
 - (Req-RoundTrips) Solution shall allow for multiple data exchanges between Sender and Receiver in the process of credential transfer. This requirement shall align with (Req-Connectivity) above.
 - (Req-ConnectionIntegrity) When Provisioning process requires multiple data exchanges (as described in Req-RoundTrips), no third party shall be allowed to interfere between Sender and Receiver. The solution shall provide integrity to the connection between Sender and Receiver for the duration of the transfer. The transfer ends when either Sender or Receiver ends it.
 - (Req-Opaque) In the case when an intermediary server is used to facilitate the credential transfer, message content between Sender and Receiver must be opaque to an intermediary, intermediary server shall not be able to recognize the content of Provisioning Information or use it to provision Digital Credential on its own.
-- (Req-Retrievals) Sender should be able to specify the max number of participant devices that can participate in the credential transfer.
+- (Req-Retrievals) : Sender should be able to specify the max number of unique Receivers that can receive Provisioning Information from the Togress solution.
```
Comment (dimmyvi) [inline draft-tigress-requirements.md:187]: 1) I think we need more details on why there might be a need for n receivers. Provide a use case?
2) Type-o in "Tigress"

Comment (ykarandikar) [reply draft-tigress-requirements.md]: Fixed the typo. Working on adding a bit more explanation.

**Comment by ykarandikar** [comment]
> I think one suggestion is to generalize this to allowing the sender to specify meta data to the provisioning partner, and that meta data could include a limit on retrievals

The limit on number of provisions is outside the scope of Tigress channel. Some use cases already have the limit and provisioning partners enforce it. This requirement is about making sure that channel is not replicating the data and letting more people retrieve it than what the Sender intends.

**Review by alexpell00 — COMMENTED** [review]
LGTM!

**Pull request merged**